docs/docs/Get-Started/get-started-quickstart.md

```diff
@@ -4,188 +4,519 @@ slug: /get-started-quickstart
-5. Click **Processing**, select the **Parser** component, and then drag it to the canvas.
-The [Parser](/components-processing#parser) component converts the data from the **Astra DB** component into plain text.
-6. Click **Embeddings**, select the **OpenAI Embeddings** component, and then drag it to the canvas.
-The [OpenAI Embeddings](/components-embedding-models#openai-embeddings) component generates embeddings for the user's input, which are compared to the vector data in the database.
-7. Modify the **Prompt** component to contain variables for both `{user_question}` and `{context}`.
-The `{context}` variable gives the bot additional context for answering `{user_question}` beyond what the LLM was trained on.
-
-```text
-Given the context
-{context}
-Answer the question
-{user_question}
+- [An OpenAI API key](https://platform.openai.com/api-keys)
+
+## Run the Simple Agent template flow
+
+1. In Langflow, click **New Flow**, and then select the **Simple Agent** template.
+
+![Simple agent starter flow](/img/quickstart-simple-agent-flow.png)
+
+The Simple Agent flow consists of an [Agent component](/components-agents) connected to [Chat I/O components](/components-io), a [Calculator component](/components-data#calculator), and a [URL component](/components-data#url). When you run this flow, you submit a query to the agent through the Chat Input component, the agent uses the Calculator and URL tools to generate a response, and then returns the response through the Chat Output component.
+
+Many components can be tools for agents, including [Model Context Protocol (MCP) servers](/mcp-server). The agent decides which tools to call based on the context of a given query.
+
+2. In the **Agent** component's settings, in the **OpenAI API Key** field, enter your OpenAI API key.
+This guide uses an OpenAI model for demonstration purposes. If you want to use a different provider, change the **Model Provider** field, and then provide credentials for your selected provider.
+
+    Optionally, you can click <Icon name="Globe" aria-hidden="true"/> **Globe** to store the key in a Langflow [global variable](/configuration-global-variables).
+
+3. To run the flow, click <Icon name="Play" aria-hidden="true"/> **Playground**.
+
+4. To test the Calculator tool, ask the agent a simple math question, such as `I want to add 4 and 4.`
+To help you test and evaluate your flows, the Playground shows the agent's reasoning process as it analyzes the prompt, selects a tool, and then uses the tool to generate a response.
+In this case, a math question causes the agent to select the Calculator tool and use an action like `evaluate_expression`.
+
+![Playground with Agent tool](/img/quickstart-simple-agent-playground.png)
+
+5. To test the URL tool, ask the agent about current events.
+For this request, the agent selects the URL tool's `fetch_content` action, and then returns a summary of current news headlines.
```

Comment:
This feels a little opaque. Where did we provide the news URL? Is it already pre-filled?

Reply:
I will have to come back to this question.